src/main/java/com/example/rets_api/controller/RetsController.java

```diff
@@ -29,8 +29,13 @@ public ResponseEntity<List<PropertyDTO>> getPropertiesByParams(@RequestBody Prop
     }
 
     @GetMapping(value="/{id}")
-    public ResponseEntity<PropertyDTO> getPropertiesById(@PathVariable("id") Long propertyId){
+    public ResponseEntity<PropertyDTO> getPropertyById(@PathVariable("id") Long propertyId){
         return ResponseEntity.ok(propertyService.getPropertyById(propertyId));
     }
 
+    @GetMapping(value = "/all")
```

Comment:
For Rest application get all don't need a endpoint, you can remove the (value = "/all"). But if you remove we'll have a conflict with the getByFilter. As for REST application the GET don't have a body, we can skip the REST patterns for this method and add some value to the GetMapping, maybe like (value = "/byFilter")

Reply:
changes done and collection updated